src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +51,161 @@ export const KafkaPage: FunctionComponent = () => {
               </CardTitle>
             </CardHeader>
-            <CardBody>{t("kafkaoverview:contactSalesCardMainText")}</CardBody>
+            <CardBody>{t("contactSalesCardMainText")}</CardBody>
             <CardFooter>
               <Button
                 data-testid="cardContactSales-buttonCTA"
-                variant={ButtonVariant.secondary}
+                variant={ButtonVariant.link}
+                isLarge
+                isInline
                 component="a"
-                href="https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-streams-for-apache-kafka#contact-sales"
+                href="https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-streams-for-apache-kafka#contact-us"
                 target="_blank"
               >
-                {t("kafkaoverview:contactSalesCardCallToActionButton")}
+                {t("contactSalesCardCallToActionButton")}
                 <ExternalLinkAltIcon className="pf-u-ml-md" />
               </Button>
             </CardFooter>
           </Card>
         </Grid>
       </MarketingPageSection>
+      <MarketingPageSection>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h3">{t("pricingModalTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Flex>
+              <Flex flex={{ default: "flex_1" }}>
+                <FlexItem>
+                  <DescriptionList
+                    isHorizontal
+                    className="description-list-spacer"
+                  >
+                    <DescriptionListGroup className="description-list-spacer">
+                      <DescriptionListTerm>
+                        {t("steamingUnit")}
+                      </DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$1.49/hour"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                    <DescriptionListGroup className="description-list-spacer">
+                      <DescriptionListTerm>
+                        {t("dataTransfer")}
+                      </DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$0.09/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                    <DescriptionListGroup className="description-list-spacer">
+                      <DescriptionListTerm>{t("storage")}</DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$0.10/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                  </DescriptionList>
+                </FlexItem>
+              </Flex>
+              <Divider isVertical />
+              <Flex flex={{ default: "flex_1" }}>
+                <FlexItem>
```

Comment:
The PF examples are somewhat misleading, in that they show this structure in a lot of their examples:
```
<Flex>
  <Flex>
    <FlexItem>
```
Most of the time, just a single `<Flex>` component is needed (and `<FlexItem>` is optional—the first descendants will be treated as flex items in the layout, similar to what's happening with `<Divider>` above. 

**For this section,** I suggest changing this:
```
              <Flex flex={{ default: "flex_1" }}>
                <FlexItem>
```
to this:
```
              <FlexItem flex={{ default: "flex_1" }}>
```

**For the first section,** however, you should not make this change. For that section, nesting an additional `<Flex>` component is necessary for using the `justifyContent` prop to center the contents horizontally.